apps/wallet/src/background/account-sources/accountSource.ts

```diff
@@ -14,6 +14,7 @@ import {
 export enum AccountSourceType {
     Mnemonic = 'mnemonic',
     Seed = 'seed',
+    Keystone = 'keystone',
 }
```

Comment:
Btw why is ledger not listed here?

Reply:
Because ledger doesnt have an account source, just accounts